src/services/Blockchain/submitTransaction.ts

```diff
@@ -6,15 +6,15 @@ import { calculateGasMargin } from 'utils'
 
 export const useSubmitTransaction = (): ((
   methodsName: string,
-  args: Array<string> | undefined,
+  args: Array<any> | undefined,
   contract: Contract | null,
   account: string | null,
   library: Web3Provider | undefined
 ) => void) => {
   return useCallback(
     async (
       methodsName: string,
-      args: Array<string> | undefined,
+      args: Array<any> | undefined,
```

Comment:
I think it could be string|umber|bignumber rather than any as it might take unsupported types
will push an update for this,